MediaBrowser.Providers/Plugins/Tmdb/TV/TmdbEpisodeProvider.cs

```diff
@@ -111,7 +111,7 @@ public async Task<MetadataResult<Episode>> GetMetadata(EpisodeInfo info, Cancell
 
                 if (response.External_Ids.Tvdb_Id > 0)
```

Comment:
```suggestion
                if (response.External_Ids.Tvdb_Id != null && response.External_Ids.Tvdb_Id > 0)
```

Missing null checks?

Reply:
A comparison with null will always return false, so it's redundant.